crates/host/src/module/mod.rs

```diff
@@ -385,16 +385,6 @@ impl ModuleCache {
     }
 }
 
-/// Configuration of a Target for wasmer for iOS
-pub fn wasmer_ios_target() -> Target {
```

Comment:
this function does not appear to be used anywhere in consuming crates

Reply:
This was added by @jost-s, was it something that used to be in Holochain core and got moved here? That's what I vaguely remember. If so, was it something that Guillem was using?

Reply:
Yes I think that's right -- I don't see it used anywhere https://github.com/search?q=wasmer_ios_target&type=code

But actually now I realized that parts of this functionality aren't working so I'm not sure it was ever really tested or used. I'm going to wait to merge this PR and get it working fully as part of it.